### ISSUE TITLE
Release 0.5

### ISSUE THREAD

**Issue by sebpuetz** [finalfusion/finalfusion-python#52]
We have added quite some features in the last month, e.g. the read methods for other formats, exposure of subword indices and the embedding similarity method.

Once #51 is done we could cut a new release, unless there is something else in the pipeline for the Python bindings.

What do you think?

**Comment by danieldk** [comment]
Agreed, once there is a new PyO3 and #51 can be solved, we could branch.

**Comment by danieldk** [comment]
Another question is: when do we release the 1.0? Also of finalfusion-rust. With this I am a bit worried that PyO3 changes may break an API (maybe we should wait until PyO3 1.0). But it would be good in both cases to have a stable API that we promise not to break. I'll make an issue in finalfusion-rust.

**Comment by danieldk** [comment]
Oh, maybe it would be nice to have support for memory-mapped quantized arrays as well. I will try to implement `WriteChunk` for mmap'ed quantized arrays today as well. We could then cut a new release of `finalfusion-rust` soonish and then base `finalfusion-python` 0.5 on that.

**Comment by danieldk** [comment]
I have created a milestone for 0.5, since this allows us to track the blocking issues:

https://github.com/finalfusion/finalfusion-python/milestone/1

**Comment by danieldk** [comment]
I am doing a final build test on AppVeyor, to check that the Windows wheels are built successfully. If so, I will branch 0.5 and tag 0.5.0.
